release-notes/6.0/preview/6.0.0-preview.4-runtime-deps.json

```diff
@@ -0,0 +1,1035 @@
+            {
+              "name": "libx11-dev",
+              "dependencyType": "LinuxPackage",
+              "usage": "xwindows"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "rid": "opensuse",
+      "components": [
+        {
+          "name": "Microsoft.NETCore.App",
+          "type": "Framework",
+          "platformDependencies": [
+            {
+              "name": "glibc",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "icu",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "krb5",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libgcc_s1",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "liblttng-ust0",
+              "dependencyType": "LinuxPackage",
+              "usage": "diagnostics"
+            },
+            {
+              "name": "libopenssl1_0_0 || libopenssl1_1",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libstdc++6",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "timezone",
+              "dependencyType": "LinuxPackage",
+              "usage": "localization"
+            },
+            {
+              "name": "zlib",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            }
+          ]
+        },
+        {
+          "name": "System.DirectoryServices.Protocols",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "libldap-2_4-2",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            }
+          ]
+        },
+        {
+          "name": "System.Drawing.Common",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "libcups2",
+              "dependencyType": "LinuxPackage",
+              "usage": "printing"
+            },
+            {
+              "name": "libgdiplus0:6.0.1",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libx11-dev",
+              "dependencyType": "LinuxPackage",
+              "usage": "xwindows"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "rid": "osx",
+      "components": [
+        {
+          "name": "Microsoft.NETCore.App",
+          "type": "Framework",
+          "platformDependencies": [
+            {
+              "name": "Core Foundation",
+              "dependencyType": "MacOSFramework",
```

Comment:
We depend on number of other macOS frameworks. Is it really useful to list them all since they all come with macOS by default?

Reply:
dtto for Windows OS .dlls

Reply:
A valid question for Mac.  For Windows, not all Windows SKUs are equal.  For example, Nano Server is missing many libraries compared to Windows Server Core or Windows Server/Desktop.  It's helpful know what our dependencies are in order to know whether things will run in one of those systems.

Reply:
> Nano Server is missing many libraries compared to Windows Server Core

It is also missing APIs in number of libraries. For example, the full `kernel32` on Window Server has more APIs than `kernel32` on Nano Server or some of the `kernel32` APIs are stubbed out.

I am not sure how much the list of Windows .dll libraries helps with determining whether given package is going work on Nano Server.

Reply:
> It is also missing APIs in number of libraries. For example, the full kernel32 on Window Server has more APIs than kernel32 on Nano Server or some of the kernel32 APIs are stubbed out.

That's fair.  However, I'm not aware of any usage of such an API where the library exists in Nano Server but not that API.  Are there any that you know of?

OTOH, there are several known instances of libraries that are not available in Nano Server which we've directly received feedback from customers about.  Examples are the following:

* System.Data.Odbc
  * odbc32.dll
* System.DirectoryServices:
  * activeds.dll
  * netapi32.dll
* System.DirectoryServices.AccountManagement:
  * activeds.dll
  * credui.dll
* System.Drawing.Common
  * gdiplus.dll

Based on that, I do see value in tracking these libraries.